app/build.gradle.kts

```diff
@@ -68,11 +68,11 @@ dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.0-RC")
     implementation("io.ktor:ktor-client-core:2.3.4")
     implementation("io.ktor:ktor-client-android:2.3.4")
-    implementation("io.github.funkatronics:multimult:0.2.0")
-    implementation("io.github.funkatronics:rpccore:0.2.1")
-    implementation("io.github.funkatronics:web3kt-solana:0.1.1")
 
-    implementation("com.solanamobile:mobile-wallet-adapter-clientlib-ktx:1.2.0")
+    implementation("com.solanamobile:web3-solana:0.2.2")
+    implementation("com.solanamobile:rpc-core:0.2.3")
+    implementation("com.solanamobile:mobile-wallet-adapter-clientlib-ktx:2.0.0")
+    implementation("foundation.metaplex:base58:0.2.10")
```

Comment:
fyi multimult (which is already being included transitively) has a Base58 decoder. among other things.

Reply:
Android Studio IDE wasn't able to detect the `Base58` import or multimult, not sure if I'm doing something wrong.